tasks/EL.yml

```diff
@@ -12,6 +12,7 @@
   when: ansible_distribution == "EL"
 
 - name: Install Wine
+  become: true
```

Comment:
The other tasks in this file (installing EPEL) also require root level access.

Reply:
you are right. I didn't noticed them. *(Probably because I am not using rpm packages often)*

It is now added and amended to this commit.